src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -0,0 +1,284 @@
+            cbsd_record = [record['registration'] for record in cbsd_dump_data if record['id'] == record_id]
+            self.assertEqual(1, len(cbsd_record))
+            # required parameters
+            self.assertEqual(device['cbsdCategory'], cbsd_record[0]['cbsdCategory'])
+            self.assertEqual(device['fccId'], cbsd_record[0]['fccId'])
+            self.assertEqual(device['airInterface']['radioTechnology'],\
+                             cbsd_record[0]['airInterface']['radioTechnology'])
+            self.assertEqual(device['installationParam']['latitude'],\
+                             cbsd_record[0]['installationParam']['latitude'])
+            self.assertEqual(device['installationParam']['longitude'],\
+                             cbsd_record[0]['installationParam']['longitude'])
+            self.assertEqual(device['installationParam']['height'],\
+                             cbsd_record[0]['installationParam']['height'])
+            self.assertEqual(device['installationParam']['heightType'], \
+                             cbsd_record[0]['installationParam']['heightType'])
+            self.assertEqual(device['installationParam']['antennaGain'], \
+                             cbsd_record[0]['installationParam']['antennaGain'])
+             # parameters should exist in record if exist in device,\
+             # if exist only in record, its value should be equal to a default value          
+            self.assertEqualToDeviceParamOrIfExistWithDefault('indoorDeployment', \
+                device['installationParam'], cbsd_record[0]['installationParam'], False)
+            
+            self.assertEqualToDeviceParamOrIfExistWithDefault('antennaAzimuth', \
+                device['installationParam'], cbsd_record[0]['installationParam'], 0)
+            
+            self.assertEqualToDeviceParamOrIfExistWithDefault('antennaDowntilt', \
+                device['installationParam'], cbsd_record[0]['installationParam'], 0)
+            
+            self.assertEqualToDeviceParamOrIfExistWithDefault('antennaBeamwidth', \
+                device['installationParam'], cbsd_record[0]['installationParam'], 0)
+            # if callSign exist, it should have the same value as registered
+            if 'callSign' in cbsd_record[0]:
+                self.assertEqual(device['callSign'], cbsd_record[0]['callSign'])
+            max_eirp_by_MHz = 37;           
+            # if eirpCapacity in the record, it should be the same as device or in accepted limits
+            if 'eirpCapability' in cbsd_record[0]:
+                if 'eirpCapability' in device:
+                    self.assertEqual(device['eirpCapability'], cbsd_record[0]['eirpCapability'])
+                    max_eirp_by_MHz = cbsd_record[0]['eirpCapability'] - 10
+                else:
+                    self.assertLessEqual(cbsd_record[0]['eirpCapability'], 47)
```

Comment:
I think you have to check that it's equal to the max value associated with that FCC ID? Or that it's missing... I'm not too sure about this. Takashi might know the right thing to do in this case.

Reply:
I think this comment is still relevant, assuming we can fill in default values. (And if not, this line should be removed.)

Reply:
sorry I missed those comments :(  I used the value 47 not because it is a "default" value, but because it is the value injected with all the Fcc IDs  in the test code.

Reply:
Ahh, I understand now. Ideally we would automatically link the two values so that it's impossible for someone to change one without changing the other, but I think that would be difficult and ultimately not worth it (you could create a global constant but that would probably cause more problems than it would solve). Since we do not expect to configure this value within the test case, I suggest leaving the implementation as is (again, assuming that we expect the SAS to fill in the default values) and add two comments, one here and one where we inject the FCC ID, to make it clear that they have to change simultaneously. What do you think?